game.py

```diff
@@ -51,18 +52,27 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, known=True, **kwargs)
 
 
-def lookup_game_by_name_or_alias(name):
-    # Name may contain extra junk, e.g. "I'd play cs later, after food" would mean name="cs later, after food"
-    for game in games:
+def lookup_known_game_by_name_or_alias(name):
+    for game in get_known_games():
         if game.loosely_matches(name):
             return game
-    return Game(name=name) if name else None
+
+def lookup_game_by_name_or_alias(name):
+    # Name may contain extra junk, e.g. "I'd play cs later, after food" would mean name="cs later, after food"
+    game = lookup_known_game_by_name_or_alias(name)
+    return game if game else Game(name=name)
 
 
-def get_known_games(json_filename='known_games.json'):
+def read_games_json_dict(json_filename=os.path.join(os.path.dirname(__file__), 'known_games.json')):
     with open(json_filename) as json_file:
-        known_game_dict = json.load(json_file)
-    return [KnownGame(name=name, **props) for name, props in known_game_dict.items()]
+        return json.load(json_file)
+
 
+def write_games_json_dict(known_games_json, json_filename=os.path.join(os.path.dirname(__file__), 'known_games.json')):
```

Comment:
Same here with the naming, also the `known_games_json` param is actually a dict so should be `known_games_dict`